conda_smithy/ci_register.py

```diff
@@ -178,48 +178,42 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/repos/{user}/{project}'.format(endpoint, user=user, project=project)
 
-    found = False
     count = 0
 
-    while not found:
+    while True:
         count += 1
 
         response = requests.get(url, headers=headers)
         try:
             response.raise_for_status()
             content = response.json()
-        except (requests.HTTPError, ValueError):
+            if "repo" in content:
+                break
+        except requests.HTTPError:
             # We regularly seem to hit this issue during automated feedstock registration on github.
             # https://github.com/conda-forge/conda-smithy/issues/233
             # ValueError: No JSON object could be decoded
             # Maybe trying again in a few seconds will fix this.
             print('travis-ci says: %s' % response.text)
-            raise
```

Comment:
Actually it looks like the `HTTPerror` itself is pretty informative or something else along these lines. Especially now that we know what is happening. Maybe we can just print the exception directly and this will be cleaner. Snippet from my IPython session below.

```python
In [1]: import requests
    ...: 
    ...: response = requests.get(
    ...:     'https://api.travis-ci.org/repos/conda-forge/foobar',
    ...:     headers={
    ...:         'Accept': 'application/vnd.travis-ci.2+json',
    ...:         'Content-Type': 'application/json',
    ...:         'User-Agent': 'Travis/1.0'
    ...:     }
    ...: )
    ...: 
    ...: try:
    ...:     response.raise_for_status()
    ...: except requests.HTTPError as e:
    ...:     print(e)
    ...:     
404 Client Error: Not Found for url: https://api.travis-ci.org/repos/conda-forge/foobar
```